alf/examples/train_ppo_pr2.py

```diff
@@ -0,0 +1,327 @@
+# coding=utf-8
+# Copyright 2018 The TF-Agents Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""Train and Eval PPO.
```

Comment:
Maybe we can add some information like:

r"""Train and Eval PR2-Gripper Env with PPO.

An example for the social-bot PR2-Gripper env using TF-Agents
The original file is from the PPO example of TF-Agents:
    https://github.com/tensorflow/agents
...
And then give a simple description about what was modified.
...

Reply:
sure.